Sources/Widgets/Representations/ArrowHandleRepresentation/index.js

```diff
@@ -332,11 +332,11 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
     ctxVisible = true,
     handleVisible = true
   ) => {
-    const state = publicAPI.getRepresentationStates()[0];
+    const hasValidState = publicAPI.getRepresentationStates().length > 0;
     superClass.updateActorVisibility(
       renderingType,
       ctxVisible,
-      handleVisible && state.isVisible()
+      handleVisible && hasValidState
```

Comment:
Just to check: we only care if there are available states, and not that the first state (if it exists) is visible, correct?